src/model.h

```diff
@@ -148,6 +148,11 @@ class Model {
     return config_.response_cache().enable();
   }
 
+  bool isDecoupled() const
```

Comment:
Upper camel case functions like the rest `IsDecoupled()`
```suggestion
  bool IsDecoupled() const
```

Reply:
Oops typo